modules/database-commons/src/test/java/org/testcontainers/ext/ScriptScannerTest.java

```diff
@@ -0,0 +1,50 @@
+package org.testcontainers.ext;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+
+import java.util.regex.Pattern;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ScriptScannerTest {
+
+    @Test
+    public void testHugeStringLiteral() {
+        String script = "/* a comment */    \"" +  StringUtils.repeat('~', 10000) + "\";";
```

Comment:
```suggestion
        String script = "/* a comment */    \"" + StringUtils.repeat('~', 10000) + "\";";
```

Reply:
Oh it was this space, not the one in the String 😬